### PR TITLE
Feature/some fixes

### PR DESCRIPTION
**Pull request by mmoreram**

### CODE REVIEW

File: src/Elcodi/Bundle/ProductBundle/composer.json [hunk 1]
```diff
@@ -48,7 +48,6 @@
         "elcodi/media-bundle": "^1.0",
         "elcodi/currency-bundle": "^1.0",
         "elcodi/attribute-bundle": "^1.0",
-        "elcodi/configuration-bundle": "^1.0",
```
Comment (EmanueleMinotto) [inline src/Elcodi/Bundle/ProductBundle/composer.json]: Why?

Comment (mmoreram) [inline src/Elcodi/Bundle/ProductBundle/composer.json]: Configuration and ConfigurationBundle are not used anymore.
Indeed, both packages are unsupported ;)

Comment (EmanueleMinotto) [inline src/Elcodi/Bundle/ProductBundle/composer.json]: Oh right, I forgot the PR https://github.com/elcodi/elcodi/pull/996 ^^'

**Comment by EmanueleMinotto** [comment]
:+1:

**Pull request merged**